Makefile

```diff
@@ -0,0 +1,11 @@
+install:
+	pipenv install --dev
+
+check:
+	PIPENV_PYUP_API_KEY="" pipenv check -i 40291
```

Comment:
Don't think we need `PIPENV_PYUP_API_KEY=""` and `-i 40291` now

Reply:
I checked... we do need it